[2025-04-12T14:22:45.648+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weekly_comment_analysis_pipeline.fetch_recent_comments manual__2025-04-12T14:22:31.796470+00:00 [queued]>
[2025-04-12T14:22:45.671+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weekly_comment_analysis_pipeline.fetch_recent_comments manual__2025-04-12T14:22:31.796470+00:00 [queued]>
[2025-04-12T14:22:45.671+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-12T14:22:45.699+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_recent_comments> on 2025-04-12 14:22:31.796470+00:00
[2025-04-12T14:22:45.717+0000] {standard_task_runner.py:60} INFO - Started process 94087 to run task
[2025-04-12T14:22:45.721+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weekly_comment_analysis_pipeline', 'fetch_recent_comments', 'manual__2025-04-12T14:22:31.796470+00:00', '--job-id', '246', '--raw', '--subdir', 'DAGS_FOLDER/Sentiment_Topicmodelling.py', '--cfg-path', '/tmp/tmppp500v5r']
[2025-04-12T14:22:45.724+0000] {standard_task_runner.py:88} INFO - Job 246: Subtask fetch_recent_comments
[2025-04-12T14:22:45.836+0000] {task_command.py:423} INFO - Running <TaskInstance: weekly_comment_analysis_pipeline.fetch_recent_comments manual__2025-04-12T14:22:31.796470+00:00 [running]> on host 09c1abf4dbfb
[2025-04-12T14:22:45.973+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weekly_comment_analysis_pipeline' AIRFLOW_CTX_TASK_ID='fetch_recent_comments' AIRFLOW_CTX_EXECUTION_DATE='2025-04-12T14:22:31.796470+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-12T14:22:31.796470+00:00'
[2025-04-12T14:22:45.993+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-04-12T14:22:46.121+0000] {base.py:83} INFO - Using connection ID 'aws_pg' for task execution.
[2025-04-12T14:22:51.294+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-12T14:22:51.330+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weekly_comment_analysis_pipeline, task_id=fetch_recent_comments, execution_date=20250412T142231, start_date=20250412T142245, end_date=20250412T142251
[2025-04-12T14:22:51.406+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-12T14:22:51.460+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
